c/include/ml-api-common.h

```diff
@@ -462,7 +462,37 @@ int ml_option_destroy (ml_option_h option);
  * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
  * @retval #ML_ERROR_INVALID_PARAMETER Fail. The parameter is invalid.
  */
-int ml_option_set (ml_option_h option, const char* key, void *value, ml_data_destroy_cb destroy);
+int ml_option_set (ml_option_h option, const char *key, void *value, ml_data_destroy_cb destroy);
+
+/**
+ * @brief A handle of machine learning information, key-value pair.
+ * @since_tizen 7.5
+ */
+typedef void *ml_info_h;
+
+/**
+ * @brief Gets a value of key in ml-info instance.
+ * @since_tizen 7.5
```

Comment:
` * @remarks The @a value should be released using free().`
This line should be added.